src/test/rustdoc/nested-modules.rs

```diff
@@ -0,0 +1,17 @@
+#![crate_name = "foo"]
+
+mod a_module {
+  fn private_function() {
+      println!("called `a_module::private_function()`");
```

Comment:
Rustdoc ignores the bodies of functions, so can you remove the `println!`s here and below?

```suggestion
```

Reply:
Hi @camelid, thanks for your didactic feedback!